src/app/builder/builder.component.html

```diff
@@ -122,32 +132,78 @@
       <h4>Copy and Use Your Embed Code</h4>
 
       <p>Copy the code for one of the following players, and place it somewhere on your webpage.</p>
+      <section class="builder-step" id="step-4a" *ngIf="props">
+          <h4>
+            <input type="checkbox" [checked]="playPlaylist" (change)="togglePlayPlaylist()">
+            Or: Make It A Playlist
```

Comment:
TBH I wasn't quite sure how to set up the builder steps to make this clearer (if there _are_ designs for the builder, I have never seen them). 

You don't need to select "play the latest episode" to get to the playlist feature; if you choose a specific episode and also choose "make it a playlist", then you'll get the player populated with the chosen ep up top, and then the X most recent episodes from your feed in the playlist below. 

I'm not quite sure how to design this best to make it most intuitive for the user. At first I had the "make it a playlist" box up near the top under the "select episode" area, but then it seemed really cluttered at the top -- and I also wasn't sure what to do about the iframe codes, whether to show all 4 options (wide single ep player, square single ep player, wide playlist player, tall playlist player) or just show the playlist players if that was selected. What do you think? Sounds like you'd prefer the "make it a playlist" to be up below the episode selection? Should the copyable iframe code change to only show the playlist options when that's selected?  


There are some designs for the playlist -- see below. 
![thumb_playist embeds](https://cloud.githubusercontent.com/assets/16007382/26502144/9d4177fc-4209-11e7-81e0-18dc0076ca40.png)